gunrock/app/hits/hits_enactor.cuh

```diff
@@ -89,14 +89,17 @@ struct hitsIterationLoop
     auto &hrank_mag = data_slice.hrank_mag;
     auto &arank_mag = data_slice.arank_mag;
 
+    auto &cur_error = data_slice.cur_error;
+
     // Set the frontier to NULL to specify that it should include
     // all vertices
-    util::Array1D<SizeT, VertexT> *null_frontier = NULL;
+    util::Array1D<SizeT, VertexT> *null_frontier = NULL; // TODO: Check that I'm doing this correctly
```

Comment:
What are you not sure you're doing correctly?

Reply:
I wasn't sure how to initialize a frontier with all nodes in the graph, but I've removed the comment since this is the correct way.